src/canmatrix/arxml.py

```diff
@@ -1200,12 +1200,22 @@ def getSignals(signalarray, Bo, xmlRoot, ns, multiplexId, float_factory):
                     newSig.addValues(1, "TRUE")
                     newSig.addValues(0, "FALSE")
 
+            def guessValue(textValue):
```

Comment:
I don't know where we are on this 'officially' but do we and `lowerCamel` or `snake_case`?  I'll add this to the refactor ticket to consider.

https://github.com/ebroecker/canmatrix/issues/236#issuecomment-435590542

Reply:
... so you preferre `snake_case` . I tried to create new features in `snake_case` but this time, I missed my intent

Reply:
Should I change the linked ticket to just state it will be 'snake_case'?  I suppose that we could instead be picking an existing coding standard/checker (black, or something configurable) and that would cover many of these questions.